drivers/usb/usbuhci_new/usbuhci.c

```diff
@@ -2490,8 +2487,8 @@ UhciInterruptNextSOF(IN PVOID uhciExtension)
             UhciResources = UhciExtension->HcResourcesVA;
             Idx = SOF_HcdTDs->Frame & UHCI_FRAME_LIST_INDEX_MASK;
 
-            InterlockedExchangePointer((PVOID)&SOF_HcdTDs->HwTD.NextElement,
-                                       (PVOID)UhciResources->FrameList[Idx]);
+            InterlockedExchange((PLONG)&SOF_HcdTDs->HwTD.NextElement,
```

Comment:
To save you the look-up: NextElement is an ULONG, and InterlockedExchangeUL is only available inside ntoskrnl.